t/mojolicious/perl_config_lite_app.conf

```diff
@@ -0,0 +1,8 @@
+use Mojo::File 'curfile';
+{
+	# Just a comment
+	foo => "bar",
+	utf => "утф",
+	file => curfile->to_string,
+	line => __LINE__, # 7
+}
```

Comment:
This file doesn't follow our formatting rules.

Reply:
If you mean indents as two space chars, I fixed that. Anything else?

Reply:
@haukex: You should run `perltidy` on all the files instead of guessing what to change. See #1370 for details, unless you want to install something like https://metacpan.org/pod/App::githook::perltidy

Reply:
Thanks, I had missed the `.perltidyrc`.